spec/features/displaying_index_posts_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+feature 'Posting to the feed more than once' do
+  scenario 'a user can post a picture to the feed and see multiple posts' do
+    visit '/'
+    click_link 'New post'
+    attach_file('post[picture]', "spec/files/images/cat.jpeg")
+    fill_in 'Caption', with: 'Meet my best friend Jaja #catsofpinstagram'
+    click_button 'Create Post'
+    click_link 'New post'
+    attach_file('post[picture]', "spec/files/images/coffee.jpeg")
+    fill_in 'Caption', with: 'Coffee is life'
+    click_button 'Create Post'
```

Comment:
If spec is testing to display posts rather than create posts... put these inside a factory bot in `background do`